src/components/dashboard/Dashboard.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import axios from 'axios';
+import { Route } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { saveUser } from '../../actions/loginActions';
+
+import Profile from './Profile';
+import Community from './Community';
+import Account from './Account';
+
+const checkSession = () => {
+  return axios.get('/api/user').then(res => res.data)
+    .catch(e => console.log(e));
+}
+
+class Dashboard extends React.Component {
+  componentDidMount() {
+    checkSession().then(user => {
+      if (user !== '') {
```

Comment:
@bonham000 I think in the next PR, we could also change this and a couple other occurrences of expressions like the one inside the if statement  to simply `if (user) { ... }` since `''` is falsy?